experimental/traffic-portal/src/app/core/ds-card/ds-card.component.html

```diff
@@ -11,15 +11,15 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<mat-card [ngClass]="{'inactive': !deliveryService.active, 'open': open, 'first': first, 'last': last}">
+<mat-card id="{{deliveryService.xmlId}}" [ngClass]="{'inactive': !deliveryService.active, 'open': open, 'first': first, 'last': last}">
 	<mat-card-title-group (click)="toggle()">
 		<mat-card-title>{{deliveryService.displayName}}{{deliveryService.active ? '' : ' (inactive)'}}
 			<a href="{{deliveryService.infoUrl}}" *ngIf="deliveryService.infoUrl" class="color-accent-inverted info" rel="noopener" target="_blank" title="More Information"><fa-icon [icon]="infoIcon"></fa-icon></a>
 		</mat-card-title>
 		<mat-card-subtitle>{{deliveryService.type ? deliveryService.type : ''}}</mat-card-subtitle>
 	</mat-card-title-group>
 	<mat-card-content class="card-content" *ngIf="open" [@enterAnimation]>
-		<div>
+		<div id="{{deliveryService.xmlId}}Content">
```

Comment:
I don't feel good about adding classes and IDs just for testing to look for - I think the card having an ID overall is actually kinda a good idea in general so that maybe people could link to a specific DS (although I imagine in general they'd link to a page where the search filters down to just that DS, but still) but this one doesn't seem to have any utility beyond just being shorter to type than `mat-card#XMLID mat-card-content > div`.